config/crd/bases/designate.openstack.org_designateapis.yaml

```diff
@@ -129,6 +129,26 @@ spec:
                 description: Resources - Compute Resources required by this service
                   (Limits/Requests). https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
                 properties:
+                  claims:
```

Comment:
@amolkahat what are these changes this seems to go beyond just fixes for precommit?

is this related to precommit?

Reply:
This changes are added by `operator-lint `.

Reply:
ok

i did some digging I think this is related to Persistent Volume Claims? I found that https://qubitpi.github.io/kubernetes-website/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-claim-v1/#PersistentVolumeClaimSpec

where it talks about resources.claims as well as the .limits and .requests that follow it